addons/website/static/src/js/menu/new_content.js

```diff
@@ -28,7 +28,18 @@ var NewContentMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
         failed: _t('Failed to install "%s"'),
         installInProgress: _t("The installation of an App is already in progress."),
         installNeeded: _t('Do you want to install the "%s" App?'),
-        installPleaseWait: _t('Installing "%s"'),
+        installPleaseWait: {
+            default: _t('Installing "%s"'),
+            eCommerce: [
+                _t("keep this page open,"),
+                _t("relax and"),
+                _t("get a coffee"),
+                _t("In ~ 20 seconds"),
+                _t("You'll be able to"),
+                _t("design beautiful"),
+                _t("product pages"),
+            ]
```

Comment:
Nevermind, website_sale is not installed yet so the include wouldn't work... Maybe we could have that data in the manifest and load it into the ir.module, but that's certainly not something we can do in stable. @qsm-odoo opinion?

Reply:
No need to loose time on this, for installation stuff, we can accept stuff like that. However, the implementation is not ok: a translator will not be able to translate parts of sentences individually and have a correct result. In french, the last part already goes

"Vous allez pouvoir"
"Designer des beaux"
"Pages de produits"

... fail. Get a language which is not structured the same way like Dutch and the sentence won't even be understandable. Maybe the spec has to be reviewed, check with the PO.

Reply:
Answer from the PO :

> I actually think that a good translator should look at the three bits and translate them so that they work together.
> Typically in French you would have;
> 'Dans 20 secs'
> 'Vous allez pouvoir'
> 'Designer votre magasin en ligne'
> 'et y mettre vos superbes produits'

https://www.odoo.com/web#active_id=2446852&cids=1&id=2446852&model=project.task&menu_id=